README.md

```diff
@@ -11,10 +11,9 @@ The 2 sections I added to build for this project are:
 
 ## Live Demo (if available)
 github repo link 
```

Comment:
[Optional] I suggest that you remove the `(if available)` note as it is only a note.